src/parsers/utils.hpp

```diff
@@ -99,6 +99,18 @@ namespace pinocchio
         throw std::invalid_argument(exception_message);
       }
     }
+    else if (string.substr(0,2).compare("./") == 0)
```

Comment:
This seems not to be robust depending on the OS. Could you use rather Boost.Filesystem for that?
Additionally, it will be nice to add a test of the fix.

Reply:
thanks for the hint. I adapted it to use `Boost.Filesystem`.